packages/back-end/src/util/secrets.ts

```diff
@@ -10,8 +10,15 @@ if (fs.existsSync(".env.local")) {
 
 export const IS_CLOUD = !!process.env.IS_CLOUD;
 
-export const UPLOAD_METHOD =
-  IS_CLOUD || process.env.UPLOAD_METHOD === "s3" ? "s3" : "local";
+export let UPLOAD_METHOD: string;
+
+if (IS_CLOUD || process.env.UPLOAD_METHOD === "s3") {
+  UPLOAD_METHOD = "s3";
+} else if (IS_CLOUD || process.env.UPLOAD_METHOD === "google-cloud") {
```

Comment:
I might refactor this to make the logic a little clearer:

```ts
export const UPLOAD_METHOD = (() => {
  if (IS_CLOUD) return "s3";

  const method = process.env.UPLOAD_METHOD;
  if (["s3", "google-cloud"].includes(method)) {
    return method;
  }

  return "local";
})();

Reply:
@jdorn Ah, thank you - I was misinterpreting the logic here. Updated.